knx/dpt/formats.go

```diff
@@ -73,3 +75,17 @@ func unpackF16(data []byte, f *float32) error {
 	*f = 0.01 * float32(m) * float32(uint(1)<<e)
 	return nil
 }
+
+func packInt(i int) []byte {
```

Comment:
This packs a `uint8`/`byte`, the name and parameter type are misleading. I suggest name `packU8` and `uint8` as parameter.